pkg/client/leaderelection/leaderelection.go

```diff
@@ -169,7 +168,6 @@ type LeaderElectionRecord struct {
 // Run starts the leader election loop
 func (le *LeaderElector) Run() {
 	defer func() {
```

Comment:
Happy to remove if you don't want it in this PR, and can follow on with a separate PR.